WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -364,32 +364,20 @@ extension UIImage {
         return UIImage(named: "woo-empty-products")!
     }
 
-    /// Empty Products Tab Icon
+    /// Message image
     ///
-    static var emptyProductsTabImage: UIImage {
-        return UIImage(named: "woo-empty-products-tab")!
-    }
-
-    /// Empty Reviews Icon
-    ///
-    static var emptyReviewsImage: UIImage {
-        return UIImage(named: "woo-empty-reviews")!
-    }
-
-    /// An image showing a hand holding a magnifying glass over a page.
-    ///
-    static var emptySearchResultsImage: UIImage {
-        UIImage(named: "woo-empty-search-results")!
+    static var messageImage: UIImage {
+        return UIImage(named: "message")!
```

Comment:
Instead of calling the images using their role in the app, I decided to use names that describe their content itself, using the exact names that we have in Figma (eyHme0rFactvLQlejGIFWT-fi-2950_16742).

I think this approach has two advantages over the previous one:
1. It encourages the developer to reuse the images when possible.
2. Keeping the same names as the ones in the Figma library will make updating them later easier.

Please share your thoughts on this.